website/docs/api/plugins/plugin-content-blog.md

```diff
@@ -59,6 +59,7 @@ Accepted fields:
 | `feedOptions.description` | `string` | <code>\`${siteConfig.title} Blog\`</code> | Description of the feed. |
 | `feedOptions.copyright` | `string` | `undefined` | Copyright message. |
 | `feedOptions.language` | `string` (See [documentation](http://www.w3.org/TR/REC-html40/struct/dirlang.html#langcodes) for possible values) | `undefined` | Language metadata of the feed. |
+| `postSortDirection` | `string` | `'desc'` | One of 'desc', 'asc'. Governs the direction of blog post sorting. |
```

Comment:
```suggestion
| `postSortDirection` | <code>'asc' \| 'desc'</code> | `'desc'` | Governs the direction of blog post sorting. |
```

Maybe also rename to `'ascending' | 'descending'`? I think it's your C idiom kicking in again 😄